Practicals/Transcriptome_Practical/Transcriptomic_Practical.Rmd

```diff
@@ -110,7 +132,7 @@ Let's check that it's installed
 whereis hisat2
 ```
 
-This should give you the output `/home/biotech7005/miniconda/bin/hisat2`
+This should give you the output `/usr/bin/hisat2`
```

Comment:
This is output from `which`, Change this to match `whereis` output or use `which` above.